server/conf/application.conf

```diff
@@ -184,7 +184,7 @@ adfs.ad_groups_attribute_name = ${?AD_GROUPS_ATTRIBUTE_NAME}
 base_url = "http://localhost:9000"
 base_url = ${?BASE_URL}
 # Staging hostname should not start with https(s)://.
-staging_hostname = "staging.seattle.civiform.com"
+staging_hostname = ""
```

Comment:
**issue (blocking):** CiviForm uses the staging hostname to detect whether not it's in staging mode, this value should be set to whatever the new domain is: https://github.com/civiform/civiform/blob/main/server/app/controllers/dev/DevController.java#L33

Reply:
Oh I just realized this is the main repo and not Seattle-specific. So long as this value is correctly set in the Seattle staging deployment code then you're good to go!